airtable_utils.py

```diff
@@ -69,17 +69,27 @@ def get_wrong_answers(hunt_password, output_file):
 
 def process_errori(error_dict, output_file):
     with open(output_file, 'w') as f_out:
-        f_out.write("ERRORI piu' frequenti (Almeno 5 volte)\n")
         for tappa in error_dict:
-            counter = collections.Counter(error_dict[tappa])
+        counter = collections.Counter(d3[tappa])
+        error_dict[tappa] = (error_dict[tappa],{'counter':sum(counter.values())})
```

Comment:
penso che ci sia qualche problema di indentazione qua

Reply:
ok, sistemeremo dopo la call in cui capiro' come fare le modifiche in locale